dotcom-rendering/src/layouts/CommentLayout.tsx

```diff
@@ -307,10 +307,11 @@ export const CommentLayout = (props: WebProps | AppsProps) => {
 		article.config.abTests.updatedHeaderDesignVariant === 'variant';
 
 	const { absoluteServerTimes = false } = article.config.switches;
-	const inTagLinkTest =
+
+	const shouldShowTagLink =
 		isWeb &&
-		article.config.abTests.tagLinkDesignVariant === 'variant' &&
-		article.tags.some((tag) => tag.id === 'football/euro-2024');
+		article.config.abTests.tagLinkDesignControl !== 'control' &&
```

Comment:
Users not in the test and users in the variant of the test will be shown the TagLink. It will not be shown to users in the control of the test.